src/components/FeedPost/FeedPost.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import {useHistory} from 'react-router-dom'
+import {useSelector} from 'react-redux'
+
+const FeedPost = ({post}) => {
+    const history = useHistory();
+
+    const state = useSelector(state => state)
+    const toProfile = (ev) => {
+        history.push(`profile/${post.user_id}`); 
+        ev.stopPropagation()
+    }
+
+    const toPost = (ev) => {
+        history.push(`post/${post._id}`)
+        ev.stopPropagation()
+    }
+    
+    if (state.users.email) {
+        return (
+            <Wrapper onClick={ev=>toPost(ev)}>
+            <Header>
```

Comment:
just a small syntax comment but as `onClick` is called with `event` parameter you could just do `onClick={toPost}` and that would work to - no need to wrap the function into an other function - this is only useful when you want to pass additional argument like i.e `onClick={(ev) => toPost(ev, props.value)}`